packs/spell-effects/spell-effect-bless.json

```diff
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Bless",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Bless]</p>\n<p>Blessings from beyond help you strike true. You gain a +1 status bonus to attack rolls.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Bless].</p>\n<p>You gain a +1 status bonus to attack rolls.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Bless]</p>\n<p>You gain a +1 status bonus to attack rolls.</p>"
```

Reply:
This is something I have ran into consistency wise not sure where stops should or shouldn't be when it comes linked things.

Reply:
We haven't been very consistent with it. Since I don't care I'll leave it to whoever the new data lead is to make the call.